catalyst/dl/experiment/experiment.py

```diff
@@ -143,6 +143,30 @@ def distributed_params(self) -> Dict:
         """Dict with the parameters for distributed and FP16 method."""
         return self._distributed_params
 
+    @property
+    def hparams(self) -> OrderedDict:
+        """Returns hyper parameters"""
+        hparams = OrderedDict()
+        if self._optimizer is not None:
+            optimizer = self._optimizer
+            hparams["optimizer"] = optimizer.__repr__().split()[0]
+            params_dict = optimizer.state_dict()["param_groups"][0]
+            for k, v in params_dict.items():
+                if k != "params":
+                    hparams[k] = v
+        loaders = self.get_loaders("train")
+        num_train_loaders = 0
+        for k in loaders.keys():
+            if "train" in k:
+                num_train_loaders += 1
+        if num_train_loaders == 1:
+            hparams["batch_size"] = loaders["train"].batch_size
+        else:
+            for k, v in loaders.items():
+                if "train" in k:
+                    hparams[f"{k}_batch_size"] = v.batch_size
```

Comment:
I think, it's better to use the same naming like this for all experiments, even if we have only 1 loader